loop_rate:    5.0 # The frequency in Hz at which to run the swarm behavior loops.
queue_size:   1   # The size of the message queue used for publishing and subscribing to topics.
max_trackers: 3   # The maximum number of UAVs that should be tracking the same target simultaneously. 0 disables check.
min_trackers: 1   # The number of UAVs tracking the same target above which a UAV stops tracking with a certain probability.
